napalm/base/helpers.py

```diff
@@ -255,3 +256,41 @@ def as_number(as_number_val):
         return (int(big) << 16) + int(little)
     else:
         return int(as_number_str)
+
+
+def canonical_interface(interface, change, short=False, update_long_to_short=None):
```

Comment:
Why not make this be?

```
def canonical_interface(interface, change=True, short=False, update_long_to_short=None):
```

Reply:
Do we even need `change` argument since we would always call this with True?

Reply:
Also you need to document the arguments here? Particularly `short` and `update_long_to_short`?

So `update_long_to_short` is a dictionary of additional names someone could feed in going from abbreviated names to full names? I don't think that name makes sense.

Reply:
np with it being `change=False`, but the idea was to not change it so drastically at first. @mirceaulinic (and I) had concerns about changing outcome so abruptly, perhaps next major version update, after we know it is good? 

Documentation will be easier once everything has been agreed upon.